src/lib/swapper/LifiSwapper/LifiSwapper.ts

```diff
@@ -67,14 +71,41 @@ export class LifiSwapper implements Swapper<EvmChainId> {
    * Builds a trade with definitive rate & txData that can be executed with executeTrade
    **/
   async buildTrade(input: BuildTradeInput): Promise<LifiTrade> {
-    return await buildTrade(input, this.lifiAssetMap, this.lifiChainMap, this.lifiBridges)
+    return await buildTrade(input, this.lifiAssetMap, this.lifiChainMap)
   }
 
   /**
    * Get a trade quote
    */
-  async getTradeQuote(input: GetEvmTradeQuoteInput): Promise<TradeQuote<EvmChainId>> {
-    return await getTradeQuote(input, this.lifiAssetMap, this.lifiChainMap, this.lifiBridges)
+  async getTradeQuote(input: GetEvmTradeQuoteInput): Promise<LifiTradeQuote> {
+    const minimumCryptoHuman = getMinimumCryptoHuman(input.sellAsset)
+    const minSellAmount = toBaseUnit(minimumCryptoHuman, input.sellAsset.precision)
+    const isBelowMinSellAmount = bn(input.sellAmountBeforeFeesCryptoBaseUnit).lt(minSellAmount)
+
+    // TEMP: return an empty quote to allow the UI to render state where buy amount is below minimum
+    // TODO: remove this when we implement monadic error handling for swapper
+    // https://github.com/shapeshift/web/issues/4237
+    if (isBelowMinSellAmount) {
+      return {
+        buyAmountCryptoBaseUnit: '0',
+        sellAmountBeforeFeesCryptoBaseUnit: input.sellAmountBeforeFeesCryptoBaseUnit,
+        feeData: {
+          networkFeeCryptoBaseUnit: '0',
```

Comment:
We might need to populate this so the gas amount is populated even when the sell amount is too low.

Reply:
as discussed, resolved in `transformLifiFeeData`